dashboard/app/controllers/aichat_controller.rb

```diff
@@ -251,4 +257,10 @@ def check_message_safety
       project_id
     end
   end
+
+  private def should_throttle?
+    id = current_user&.id || session.id
+    limit = DCDO.get('aichat_request_limit_per_min', DEFAULT_REQUEST_LIMIT_PER_MIN)
+    Cdo::Throttle.throttle(AICHAT_PREFIX + id.to_s, limit, 60)
```

Comment:
I see that the throttle error is logged via metrics reporter. I believe we log to HoneyBadger for Azure and profanity throttle errors. Would it still be useful to do so here as well?

Reply:
Good question - my understanding is that we should only log to HB for unexpected errors that we actively want to reduce (ex. our AI models should not be generating profanity, so if they are we need to fix something). In the case of throttling, there's nothing more for us to do, i.e. the system is working as expected? We will have logging on the front end to identify the root cause like you mentioned